CHANGELOG.md

```diff
@@ -1,3 +1,76 @@
+# Thrust 1.11.0
+
+## Summary
+
+Thrust 1.11.0 is a minor release providing bugfixes and performance
+enhancements.
```

Comment:
> Thrust 1.11.0 is a minor release providing bugfixes and performance enhancements.

We usually only use the phrase "minor release" to refer to a point release.

The addition of the new radix sort backend definitely makes this a major release of CUB. I think it counts as a major release for Thrust as well. 

The changelog is partially marketing; we want to show off all we've done.

Reply:
Sounds good -- updated both changelogs.